ui/nuxeo-user-group-management/nuxeo-create-user.js

```diff
@@ -40,92 +41,80 @@ import './nuxeo-edit-password.js';
   class CreateUser extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
     static get template() {
       return html`
-    <style>
-      :host {
-        display: block;
-      }
+        <style include="nuxeo-button-styles">
+          :host {
+            display: block;
+          }
 
-      .header {
-        @apply --layout-horizontal;
-        @apply --layout-center;
-        margin-bottom: 16px;
-      }
+          .header {
+            @apply --layout-horizontal;
+            @apply --layout-center;
+            margin-bottom: 16px;
+          }
 
-      .header > iron-icon {
-        padding: 8px;
-      }
+          .header > iron-icon {
+            padding: 8px;
+          }
 
-      .form-buttons {
-        margin-top: 16px;
-      }
+          .form-buttons {
+            margin-top: 16px;
+          }
 
-      #errors {
-        color: red;
-        margin: 1em 0;
-      }
+          #errors {
+            color: red;
```

Comment:
We could take the chance to align the colors here: https://nuxeowiki.atlassian.net/wiki/spaces/DT/pages/890536019/Color#Feedback

Reply:
sounds good, will sync with Carolina and see how much we can do here.